src/backend/app/db/db_models.py

```diff
@@ -129,7 +130,10 @@ class DbProject(Base):
         ),
     )
     author = relationship(DbUser, uselist=False, backref="user")
-
+    final_output = cast(
```

Comment:
this is the multiselect option.  We might need an array field ?

Reply:
I’ve updated the schema & field to use an array field to better handle multiple selections.